src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -107,59 +109,76 @@ export default class Result extends React.Component {
 
     return (
       <div>
-        {this.renderToggle()}
+        <Flex align="center" mb={space(2)}>
+          <Box flex="1">
+            <Heading>Result</Heading>
+          </Box>
+          <Box justifySelf="flex-end">{this.renderToggle()}</Box>
+        </Flex>
 
         {view === 'table' && <Table data={data} query={query} />}
         {view === 'line' && (
-          <LineChart
-            series={basicChartData}
-            height={300}
-            options={{
-              tooltip: Tooltip({
-                formatter: formatTooltip,
-              }),
-            }}
-          />
+          <Panel>
```

Comment:
There's `app/components/charts/panelChart` that we're using for health if we wanted a similar panel.

Reply:
Good idea! I'll use that instead.